### ISSUE TITLE
Add option to skip question

### ISSUE THREAD

**Issue by BieleckiLtd** [KristofferStrube/EditorConfigWizard#14]
I noticed some answers do not have a default mark, and I don’t have a preference either (like question about local static methods). I would like to be able to skip such questions.

**Comment by KristofferStrube** [comment]
Good idea. Is this something you want to contribute? Else I can add this feature.
